Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -2028,16 +2028,16 @@ public static bool IsItemStackable(DaggerfallUnityItem item)
         /// <summary>
         /// Allows loot found in containers and enemy corpses to be modified.
         /// </summary>
-        /// <param name="lootItems">An array of the loot items</param>
-        /// <returns>The number of items modified.</returns>
-        public static int ModifyFoundLootItems(ref DaggerfallUnityItem[] lootItems)
+        /// <param name="lootItems">An array of the loot items to modify</param>
+        /// <returns>The array of modified loot items</returns>
+        public static DaggerfallUnityItem[] ModifyFoundLootItems(ref DaggerfallUnityItem[] lootItems)
```

Comment:
You might want to remove the `ref` here to reflect how the function is meant to be used

Reply:
Yeah, agreed. I'll make this change after merging now this method is purposed a little differently. Provided no objections @ajrb of course? :)